mobile/src/lib/media.js

```diff
@@ -49,21 +64,7 @@ export const getPhotos = async () => {
         (response) => resolve(response.library),
         (err) => {
           console.warn(err)
-          if (err.startsWith('Permission')) {
-            requestAuthorization().then(authorization => {
```

Comment:
We don't want to ask permission when the backup starts but only when the user makes changes in the interface.
PS: we are on getPhotos function